AdminCore.ScheduledJobs/PruneExpiredEventsJob.cs

```diff
@@ -0,0 +1,73 @@
+﻿using AdminCore.Constants.Enums;
+using AdminCore.DAL.Models;
+using Quartz;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AdminCore.ScheduledJobs
+{
+  internal class PruneExpiredEventsJob : BaseDatabaseJob, IJob
+  {
+    public Task Execute(IJobExecutionContext context)
+    {
+      var schedules = DatabaseContext.SchedulesRepository.Get(x => x.IsActive);
```

Comment:
This returns an empty collection if nothing is found. You'll need to check the count.